spec/services/course/user_invitation_service_spec.rb

```diff
@@ -310,6 +312,19 @@ def invite
           end
         end
       end
+
+      context 'when no roles are specified' do
+        let(:user_attributes_without_role) do
+          user_attributes.map { |attr| attr.except(:role) }
+        end
+
+        it 'defaults to :student for roles' do
+          result_new, _, result_existing = subject.send(:invite_users, user_attributes_without_role)
+          (result_new + result_existing).each do |invit|
+            expect(invit.role).to eq('student')
```

Comment:
Optional, but after declaring enums you can do the check `invitation.student?`, for truthy or falsey value.